Translation/en/Text/Main/JapToWesternNumbers.txt

```diff
@@ -0,0 +1,135 @@
+//Just a flat counter from 00 to 99, because jap numberes were bugging me
```

Comment:
If TRR has ReplaceWideCharacters enabled this is already done in a more optimal way